assets/queries/cloudFormation/s3_bucket_versioning_not_enabled/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {
+   
+  resource := input.document[i].Resources[name]
+  
+  not resource.Properties.VersioningConfiguration.Status == "Enabled"  
+  
+  
+  result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	     sprintf("Resources.%s.Properties.VersioningConfiguration.Status", [name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": "S3 bucket versioning is enabled",
+                "keyActualValue": 	"S3 bucket versioning is not enabled"
+            }
+}
```

Comment:
Query should be divided into two. This is correct, however, if you divide into one query that checks if 'VersioningConfiguration' is undefined and another that checks, when defined, if 'Status' is set to "Suspended". This way, you can be more specific in the type of issue, because when 'undefined' it should be 'MissingAttribute' and when 'Suspended' it should be 'IncorrectValue'.